_sass/components/buttons.scss

```diff
@@ -3,7 +3,7 @@
 //------------------------------------------------
 .button {
     padding: 8px 18px;
-    font-size: $font-size-small;
+    font-size: $font-size-large;
```

Comment:
Isn’t that going to have an impact on a lot of other places?

Reply:
no its only used as a class on the front page